backup.py

```
@@ -178,19 +237,28 @@ def backup_container_appdata(source_path, dest_root, container_id, host, ssh_use
 
     if not src_path.exists():
         raise FileNotFoundError(f"Backup path does not exist: {src_path}")
 
     try:
+        if host != "local":
+            mkdir_cmd = ["ssh", "-o", "BatchMode=yes", "-p", str(ssh_port)]
+            if ssh_key:
+                mkdir_cmd.extend(["-i", ssh_key])
+            mkdir_cmd.append(f"{ssh_user}@{host}")
+            mkdir_cmd.append(f"mkdir -p '{dest_path}'")
+            subprocess.run(mkdir_cmd, check=True, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)